.gitignore

```diff
@@ -0,0 +1,4 @@
+dist
+target
+node_modules
+.idea/
```

Comment:
tip: you can place that line in a global `.gitignore` ([see](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html)) and never have to put it into a project again ✌️